Source/Core/VideoCommon/VideoUtils.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <string>
+#include <vector>
+
+namespace X11Utils
+{
+class XRRConfiguration;
+}
+
+namespace VideoUtils
+{
+std::vector<std::string> GetAvailableResolutions(X11Utils::XRRConfiguration* xrr_config);
+std::vector<std::string> GetAvailableAntialiasingModes(int& m_msaa_modes);
```

Comment:
I mean that it's not used anywhere in this commit, so it's out-of-place when reviewing :)